suite/runsuite_wrapper.pl

```diff
@@ -450,6 +450,7 @@
             $ignore_failures_64{'code_api|client.drx_buf-test'} = 1;
             $ignore_failures_64{'code_api|sample.memval_simple'} = 1;
             $ignore_failures_64{'code_api|client.drreg-test'} = 1;
+            $ignore_failures_64{'code_api|linux.signal_racesys'} = 1;
```

Comment:
Add the issue number in a comment: ` # i#731`.

Reply:
Done.